plugins/services/src/js/components/modals/CreateServiceModalForm.js

```diff
@@ -140,6 +140,10 @@ class CreateServiceModalForm extends Component {
   }
 
   componentWillReceiveProps(nextProps) {
+    if (nextProps.expandAdvancedSettings) {
+      this.forceUpdate();
```

Comment:
This is needed, otherwise the form won't get re-rendered and the advanced settings can't expand.

Reply:
Please find another solution, we have just finished removing those. Not to start adding them again